ddp-workspace/projects/ddp-sdk/src/lib/components/activityForm/activity.component.ts

```diff
@@ -87,6 +89,38 @@ export class ActivityComponent extends BaseActivityComponent implements OnInit,
             this.isScrolled = false;
         }
     }
+    // Navigate through fields
+    protected nextField(event: Event, {elements, preventDefault = false, carousel = false}: NavigateFields) {
+        preventDefault && event.preventDefault();
+        if(elements && elements.length > 0) {
+            // All provided elements array, excluding those, which doesn't have id
+            const elementsArray: Element[] = Array.from(document.querySelectorAll(elements.join(','))).filter(input => input.id && (input as HTMLElement)?.style.display !== 'none');
+            // Map data structure for elements in order to easily find them
+            const elementsObjects: Map<Element, number> = new Map(elementsArray.map((input, i) => [input, i]));
+            // Pressed element in a DOM
+            const pressedElement = event.target as Element;
+            // One-sided or Carousel behaviour while moving through fields
+            this.switchBehaviour(elementsArray, elementsObjects.get(pressedElement), carousel)
+        }
+    }
+    // Decide behaviour while navigation
+    private switchBehaviour(elements: Element[] | HTMLElement, elementIndex: number | undefined, carousel:boolean) {
+        if(carousel){
+            if(((elements as Element[]).length - 1 === elementIndex)) this.carouselRevert = true;
+            if(elementIndex === 0) this.carouselRevert = false;
+
+            this.carouselRevert ? this.moveBack(elements, elementIndex) : this.moveToNextField(elements, elementIndex);
+        } else {
+            this.moveToNextField(elements, elementIndex)
+        }
+    }
+
+    private moveToNextField(elements: Element[] | HTMLElement, elementIndex: number | undefined) {
+       elementIndex > -1 && elementIndex !== undefined ? elements[elementIndex + 1]?.focus() : elements[0]?.focus();
```

Comment:
1. you can skip `&& elementIndex !== undefined` condition because `undefined > -1 === false` in JS. 

The right part of the `elementIndex > -1 && elementIndex !== undefined` condition will never be checked.